app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+from typing import Any
+
+
+class Animal:
+    alive = []
```

Comment:
The `alive` list is a class variable, meaning it is shared among all instances of `Animal`. If you intend for each `Animal` to have its own list of `alive` instances, consider making it an instance variable instead.